test/e2e/tests/bridge/bridge-test-utils.ts

```diff
@@ -119,9 +119,7 @@ export const getBridgeFixtures = (
   }
 
   return {
-    driverOptions: {
-      openDevToolsForTabs: true,
```

Comment:
@micaelae @BZahory I noticed that this option was added in https://github.com/MetaMask/metamask-extension/pull/25044

Could you clarify what this option was used for? I am wondering whether `responsive` or `openDevToolsForTabs` is what you want here, or whether this is needed at all.

And the same question foes for the usage in `test/e2e/tests/swap-send/swap-send-test-utils.ts` (added here: https://github.com/MetaMask/metamask-extension/pull/23703)